python/paddle/fluid/log_helper.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 import logging
+from . import core  # add this for test
```

Comment:
<img width="826" alt="image" src="https://github.com/PaddlePaddle/Paddle/assets/15810355/a8b51bed-17a7-43a9-86ca-010df12ab35f">

Info above raised for this change